src/fastify-plugins/maps.js

```diff
@@ -19,9 +21,39 @@ export const FALLBACK_MAP_PREFIX = 'fallback'
 /** @type {FastifyPluginAsync<MapsPluginOpts>} */
 export async function plugin(fastify, opts) {
   if (opts.customMapPath) {
+    const { customMapPath } = opts
+
+    fastify.get(`/${CUSTOM_MAP_PREFIX}/info`, async () => {
+      const baseUrl = new URL(fastify.prefix, fastify.listeningOrigin)
+
+      if (!baseUrl.href.endsWith('/')) {
+        baseUrl.href += '/'
+      }
+
+      const style = await (
+        await fetch(new URL(`${CUSTOM_MAP_PREFIX}/style.json`, baseUrl))
+      ).json()
+
+      const stats = await fs.stat(customMapPath)
```

Comment:
should catch an error and if EOENT then 404 maybe? Shouldn't happen because the route above would 404 before here, but for completeness...